apps/frontend/src/pages/access-management/libs/helpers/transform-user-data.helper.ts

```diff
@@ -0,0 +1,12 @@
+import { formatDate } from "~/libs/helpers/helpers.js";
+
+import { type User } from "../types/types.js";
+
+const transformUserData = (users: User[]): User[] => {
```

Comment:
Let's remove this and pass the data as it's returned. This data is only used for this table, so if we wanted to re-map it on the frontend we should've done it on API level, not inside the components. But in our case we don't remap data returned from the backend and because this data is only supposed to be used for this table, we can restrict properties on the backend side if needed or pass all the fields from usual entity. So what I mean here, that in this case I would just remove this mapper and pass data directly

Reply:
I'm not sure if I understood correctly, but I tried to correct it